connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala

```diff
@@ -501,6 +505,21 @@ object KustoDataSourceUtils {
     SinkParameters(writeOptions, sourceParameters)
   }
 
+  private def getIngestionProperties(
+      isStreamingIngestion: Boolean,
+      mayBeIngestionPropertiesAsJson: Option[String]): Option[SparkIngestionProperties] = {
+
+    mayBeIngestionPropertiesAsJson match {
```

Comment:
nit, here and elsewhere:
mayBeIngestionPropertiesAsJson -> maybeIngestionPropertiesAsJson